include/crm/common/output.h

```diff
@@ -726,6 +726,11 @@ G_GNUC_NULL_TERMINATED;
         out->begin_list(out, NULL, NULL, title);          \
     }
 
+#define PCMK__OUTPUT_LIST_FOOTER(out, rc) \
+    if (rc == pcmk_rc_ok) { \
+        out->end_list(out); \
+    }
+
```

Comment:
Let's separate the PCMK__OUTPUT_LIST_FOOTER() change into its own commit

Reply:
I don't know how those two things ended up in the same patch.  I'll split them apart.